packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.routing.js

```diff
@@ -1,27 +1,35 @@
 import get from 'lodash/get';
 import { GUIDES } from './interfaces.constants';
+import { redirectTo } from './ola-pending-task/ola-pending-task.routing';
 
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('app.dedicated.server.interfaces', {
-    url: '/interfaces?:configStep',
+    url: '/interfaces',
     views: {
       'tabView@app.dedicated.server': {
         component: 'dedicatedServerInterfaces',
       },
     },
     translations: { value: ['.'], format: 'json' },
-    params: {
-      configStep: { dynamic: true },
-    },
+    redirectTo,
     resolve: {
       alertError: /* @ngInject */ ($timeout, $translate, Alerter) => (
         translateId,
         error,
+        appendError = false,
       ) =>
         $timeout(() => {
           Alerter.set(
             'alert-danger',
-            $translate.instant(translateId, { error: error.message }),
+            appendError
+              ? `${$translate.instant(translateId)}<br />${get(
```

Comment:
Not too fond of adding HTML that way, here it's ok as it's just one tag but it can quickly become hard to maintain